packages/flutter_tools/lib/src/artifacts.dart

```diff
@@ -121,17 +121,17 @@ class ArtifactStore {
       hostPlatform: HostPlatform.mac
     ),
     const Artifact._(
-      name: 'Sky Viewer',
-      fileName: 'sky_viewer.mojo',
-      category: _kViewerCategory,
-      type: ArtifactType.viewer,
+      name: 'Flutter for Mojo',
+      fileName: 'flutter.mojo',
+      category: _kShellCategory,
```

Comment:
Yeah, the category thing was a mistake.  I've been moving everything over to the shell category with the intent of killing categories whenever we change buckets.